### PR TITLE
popmenu button padding

### PR DESCRIPTION
**Pull request by MasterFloat**
Added some top and bottom padding to popupmenu's button so that when you hover on a popupmenu button the top and bottom padding goes to 0 and the border compensates for that, so that the button's text stays in it's position.

### CODE REVIEW

**Comment by Zarel** [comment]
Is this a problem for you? What browser are you using?

File: style/client.css [hunk 1]
```diff
@@ -429,7 +429,7 @@ p.or:after {
 	font-size: 8pt;
 	font-family: Verdana, Helvetica, Arial, sans-serif;
 	margin: 0 0 0 6px;
-	padding: 0 4px;
+	padding: 1 4px;
```
Comment (Zarel) [inline style/client.css]: This is against spec; you probably mean `1px`, not `1`.

**Comment by MasterFloat** [comment]
Mozilla

**Comment by Zarel** [comment]
Buttons are specced to be `box-sizing: border-box` and this button has a specified height, so in theory the vertical padding should be unnecessary.

**Comment by Zarel** [comment]
Are you using a really old version of Mozilla? If this fixes it, I guess there's no harm in it, but at least fix the spec issue.

**Comment by MasterFloat** [comment]
no, I downloaded it last year.

**Comment by Zarel** [comment]
Can you squash these changes?

**Comment by MasterFloat** [comment]
What do you mean by squashing?

**Comment by MasterFloat** [comment]
This is not a big problem, it just makes it neater.

**Comment by Zarel** [comment]
Combine the two changes in this pull request into a single pull request. It involves using the `squash` option in `git rebase -i`.

**Comment by MasterFloat** [comment]
I'm new to github. I don't really know how I can do that or what/where git rebase -i is.

**Comment by MasterFloat** [comment]
should I make another pull request, while you close this one to put those two changes?

**Comment by Zarel** [comment]
No, do **not** make another pull request. Googling "git squash" should give you some guides on how to do it.

**Comment by MasterFloat** [comment]
Ok.

**Comment by MasterFloat** [comment]
I didn't use my command prompt to make this commit, cause I don't know how to.
So it looks like I won't be able to squash my two commits, sorry Zarel.

**Comment by sirDonovan** [comment]
@MasterFloat The <a href="https://windows.github.com/">GitHub client</a> will work for rebasing/squashing even if you didn't originally use it to push the commit.

**Comment by MasterFloat** [comment]
SO, this means I can squash without any cmd?

**Comment by MasterFloat** [comment]
Yeah, I'm clueless. Sorry :[

**Comment by Zarel** [comment]
Merged in 5a5f8e0c9a6a26.

**Pull request closed without merging**